bin/setup

```diff
@@ -34,6 +34,8 @@ Dir.chdir APP_ROOT do
   run "cp keys/equifax_rsa.example keys/equifax_rsa"
   run "cp keys/equifax_rsa.pub.example keys/equifax_rsa.pub"
   run "cp certs/saml.crt.example certs/saml.crt"
+  run "gpg --dearmor < keys/equifax_gpg.pub.example > keys/equifax_gpg.pub.bin"
+  system "gpg --batch --import keys/equifax_gpg.example"
```

Comment:
so I think that the importing *is* needed, but only for decrypting, in reading the manpage, looks like there is no way to point at private keys:

```
       --secret-keyring file
              This is an obsolete option and ignored.  All secret keys are
              stored  in the `private-keys-v1.d' directory below the GnuPG
              home directory.
```